landing-page/index.html

```diff
@@ -1042,32 +1042,32 @@
             class="grid grid-cols-2 gap-6 text-[18px] font-light leading-[20px] md:grid-cols-6"
           >
             <li class="flex flex-col items-center justify-end gap-4">
-              <img src="./img/react.png" alt="React" class="h-16" />
+              <img src="./img/react.png" alt="" class="h-16" />
               <span class="text-center">React</span>
```

Comment:
Fix an a11y issue, this is a decorative image, it must have an empty alt, https://www.w3.org/WAI/tutorials/images/decorative/